utils/app/importExport.ts

```diff
@@ -24,41 +43,48 @@ export function isExportFormatV4(obj: any): obj is ExportFormatV4 {
   return obj.version === 4;
 }
 
-export const isLatestExportFormat = isExportFormatV4;
+export function isExportFormatV5(obj: any): obj is ExportFormatV5 {
+  return obj.version === 5;
+}
+
+export const isLatestExportFormat = isExportFormatV5;
 
 export function cleanData(data: SupportedExportFormats): LatestExportFormat {
+  if (isChatGPTDataFormat(data)) {
+    const converted = convertChatGPTDataToNativeFormat(data);
+    return converted;
+  }
+
+  let originalDataFormat: string | null = null;
+  // Convert V1 to V2
   if (isExportFormatV1(data)) {
-    return {
-      version: 4,
-      history: cleanConversationHistory(data),
-      folders: [],
-      prompts: [],
-    };
+    originalDataFormat = '1';
+    data = convertV1ToV2(data);
   }
 
+  // Convert V2 to V3
   if (isExportFormatV2(data)) {
-    return {
-      version: 4,
-      history: cleanConversationHistory(data.history || []),
-      folders: (data.folders || []).map((chatFolder) => ({
-        id: chatFolder.id.toString(),
-        name: chatFolder.name,
-        type: 'chat',
-      })),
-      prompts: [],
-    };
+    originalDataFormat = '2';
+    data = convertV2ToV3(data);
   }
 
-  if (isExportFormatV3(data)) {    
-    return {...data, version: 4, prompts: []};
+  // Convert V3 to V4
+  if (isExportFormatV3(data)) {
+    originalDataFormat = '3';
+    data = convertV3ToV4(data);
   }
 
-  if(isExportFormatV4(data)){
-    return data;
+  // Convert V4 to V5
+  if (isExportFormatV4(data)) {
+    originalDataFormat = '4';
+    data = convertV4ToV5(data);
   }
 
-
-  throw new Error('Unsupported data format');
+  if (!originalDataFormat) {
+    throw new Error('Unsupported data format');
+  } else {
+    return data;
+  }
 }
```

Comment:
epxorting v5 data is susscessful, but import v5 data will raise Error.
<img width="944" alt="image" src="https://user-images.githubusercontent.com/32943414/229953868-e22aa908-8901-4dc3-ade3-bac5f8984621.png">

Reply:
I tried adding those code between line 82 and 83, and it's work:
```typescript
  if (isExportFormatV5(data)) {
    originalDataFormat = '5';
  }
```

Reply:
Okay cool I'll check this out, thank you!